dockerfiles/site/website/compile.sh

```diff
@@ -39,6 +39,8 @@ git checkout $SHA1
 
 # Compilation
 
+export MAVEN_OPTS="-Xmx7168m -Xms2048m -XX:MaxPermSize=256m -XX:+UseConcMarkSweepGC -XX:-UseGCOverheadLimit"
```

Comment:
I tried and yes, maven 3.5 does not support on java 6. Here is the error:
`Exception in thread "main" java.lang.UnsupportedClassVersionError: org/apache/maven/cli/MavenCli : Unsupported major.minor version 51.0`